GenderPayGap.WebUI/Views/ChangeContactPreferences/ChangeContactPreferences.cshtml

```diff
@@ -0,0 +1,76 @@
+﻿@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@model GenderPayGap.WebUI.Models.Account.ChangeContactPreferencesViewModel
+
+@{
+    ViewBag.Title = "Change your contact preferences - Gender pay gap service";
+    Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";
+}
+
+@section BeforeMain {
+    @{
+        var crumbs = new List<CrumbViewModel>
+        {
+            new CrumbViewModel
+            {
+                Text = "Manage Account",
+                Href = Url.Action("ManageAccount", "ManageAccount", new {Area = "Account"})
+            },
+            new CrumbViewModel
+            {
+                Text = "Contact Preferences"
+            }
+        };
+    }
+
+    @(Html.GovUkBreadcrumbs(new BreadcrumbsViewModel
+    {
+        Crumbs = crumbs
+    }))
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        @(Html.GovUkErrorSummary())
+        
+        <h1 class="govuk-heading-xl">
+            Change your contact preferences
+        </h1>
+
+        <form method="POST" action="@Url.Action("ChangeContactPreferencesPost", "ChangeContactPreferences")">
+            @(Html.AntiForgeryToken())
+            
+            @(Html.GovUkCheckboxItemFor(
+                m => m.AllowContact,
+                labelOptions: new LabelViewModel
+                {
+                    Text = "I'm happy to be contacted for feedback on this service and take part in gender pay gap surveys"
+                }
+            
+            ))
+            
+            @(Html.GovUkCheckboxItemFor(
+                m => m.SendUpdates,
+                labelOptions: new LabelViewModel
+                {
+                    Text = "I would like to receive information about webinars, events, and new guidance"
+                }
+            ))

```

Comment:
These two options are the other way round on:
* the "create an account" page
* the "manage account" page

..so I think we should swap the order of them here